packages/cdk8s/src/api-object.ts

```diff
@@ -34,6 +35,27 @@ export interface ApiObjectOptions {
 }
 
 export class ApiObject extends Construct {
+  /**
+   * Returns the `ApiObject` named `Default` which is a child of the given
```

Comment:
Should this be changed to:

```suggestion
   * Returns the `ApiObject` named `Resource` which is a child of the given
```

?

https://github.com/awslabs/cdk8s/issues/373